cmd/containerd/command/main.go

```diff
@@ -274,4 +274,13 @@ func dumpStacks() {
 	}
 	buf = buf[:stackSize]
 	logrus.Infof("=== BEGIN goroutine stack dump ===\n%s\n=== END goroutine stack dump ===", buf)
+
+	// Also write to file to aid gathering diagnostics
+	name := filepath.Join(os.Getenv("TEMP"), fmt.Sprintf("containerd.%d.stacks.log", os.Getpid()))
```

Comment:
I think `os.TempDir()` is more platform agnostic as `TEMP` is likely unset on Linux.

Reply:
@estesp Sounds good. Updated. `os.TempDir` equates to `syscall.GetTempPath` on Windows, or `Getenv("TMPDIR")` and a fallback to `/tmp` on linux.